hack/testdata/pod-with-large-name.yaml

```diff
@@ -8,4 +8,4 @@ metadata:
 spec:
   containers:
   - name: kubernetes-serve-hostname
-    image: gcr.io/kubernetes-e2e-test-images/serve-hostname-amd64:1.1
+    image: gcr.io/kubernetes-e2e-test-images/serve-hostname-amd64:1.3
```

Comment:
putting the reference updates into a separate commit might make this easier to cherrypick to release branches.

Reply:
Thanks @ixdy. Will move the updates to a separate commit.